app/views/devise/registrations/edit.html.erb

```diff
@@ -7,6 +7,13 @@
   <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
     <%= render "devise/shared/error_messages", resource: resource %>
 
+    <div class="field mb-3">
+      <%= image_tag(resource.avatar_thumbnail) if resource.avatar.attached? %>
```

Comment:
What happens if there is not attached image?

Reply:
@RathanKumar there always will be an image attached by default, whenever a new user signin, so I don't think that case will ever happen. But still I'm checking if the user has avatar attached, while displaying the image, if not then there won't be any image html element created in that case.